portal/src/test/scala/cromwell/pipeline/datastorage/dao/repository/UserRepositoryTest.scala

```diff
@@ -50,6 +51,19 @@ class UserRepositoryTest extends AsyncWordSpec with Matchers with BeforeAndAfter
       }
     }
 
-  }
+    "getUsersByEmail" should {
 
+      "should find newly added user by email pattern" taggedAs (Dao) in {
+        val newUser: User = TestUserUtils.getDummyUser()
+        userRepository
+          .addUser(newUser)
+          .flatMap(
+            _ =>
+              userRepository
+                .getUsersByEmail(newUser.email)
+                .map(repoResp => repoResp should contain theSameElementsAs Vector(newUser))
```

Comment:
1. Please introduce value at line 61.
2. Why do you use Vector here?

Reply:
1. Why do I need it here, if I do not use it anywhere? 
2. changed type to Seq